yscope-k8s/README.md

```diff
@@ -0,0 +1,139 @@
+# Setup local K8s cluster for presto + clp
+
+## Install docker
+
+Follow the guide here: [docker]
+
+## Install kubectl
+
+`kubectl` is the command-line tool for interacting with Kubernetes clusters. You will use it to
+manage and inspect your k3d cluster.
+
+Follow the guide here: [kubectl]
+
+## Install k3d
+
+k3d is a lightweight wrapper to run k3s (Rancher Lab's minimal Kubernetes distribution) in docker.
+
+Follow the guide here: [k3d]
+
+## Install Helm
+
+Helm is the package manager for Kubernetes.
+
+Follow the guide here: [helm]
+
+# Launch clp-package
+1. Find the clp-package for test on our official website [clp-json-v0.4.0]. We also put the dataset for demo here: `mongod-256MB-presto-clp.log.tar.gz`.
+
+2. Untar it.
+
+3. Replace the content of `etc/clp-config.yml` with the following (also replace the IP address `${REPLACE_IP}` with the actual IP address of the host that you are running the clp-package):
```

Comment:
Can we put this file somewhere, then use a oneliner command to get and replace the ip address